python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S5439.json

```diff
@@ -3,30 +3,10 @@
   "type": "VULNERABILITY",
   "status": "ready",
```

Comment:
shouldn't it be deprecated?

Reply:
Indeed...Thanks!
I trusted Jira status when regenerating metadata and didn't double check, fixed in 86791d7aa73d423c932b6b893762b2663b448879